ui/topSection/components/defenseBlock.pug

```diff
@@ -2,8 +2,8 @@
 .defense-block
   .defenses.epic-table-header-grid
     div Def.
-    div Ability
-    div.armor-header.epic-tooltip Armor
+    div Armor
+    div.armor-header.epic-tooltip None
```

Comment:
This will probably help with understanding the two columns.

Reply:
A nice step forward!
It might make sense to get rid of defenseBoost completely. It seldom gets used, and it can cause confusion. People seem to be OK with just remembering their penalties. (I still think the sheet should figure in the weight penalty.)
With that line gone, that is half way to making room for Resolve and Robustness in the defense block.

Reply:
Actually, I use this field whenever I am tracking things like stone skin, or if a character or monster is affected by a critical failure.
I also use it in cases where someone has buffed my defense.
Specifically, I often use this:
![image](https://github.com/user-attachments/assets/5c8c90f3-ecdf-413b-99b8-cf2109db08f3)

I want to remove this:
![image](https://github.com/user-attachments/assets/60e6dddf-41ff-409a-9ddd-c03cbf22f50b)